extensions/amp-analytics/0.1/vendors.js

```diff
@@ -541,6 +544,73 @@ export const ANALYTICS_CONFIG = {
       'image': true,
     },
   },
+
+  'mpulse': {
+    'requests': {
+      'onvisible': 'https://${beacon_url}?' +
```

Comment:
I thought this was something that is specific to each publisher and hence not specified here.